shell/browser/ui/window_icon_utils_mac.mm

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2014 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/browser/ui/window_icon_utils.h"
+
+#import <Cocoa/Cocoa.h>
+#include "base/apple/foundation_util.h"
+#include "base/apple/scoped_cftyperef.h"
+#include "ui/gfx/image/image.h"
+#include "ui/gfx/image/image_skia.h"
+#include "ui/gfx/image/image_skia_util_mac.h"
+
+gfx::ImageSkia GetWindowIcon(content::DesktopMediaID id) {
```

Comment:
Can you add a comment explaining why we copy code from Chromium here and a link to the original source code that this copies?

You can add the link similar to the example comment from https://github.com/electron/electron/issues/48063#issuecomment-3649950542.

Reply:
Can you link to the code at the specific commit that you copied from?

<img width="690" height="360" alt="Image" src="https://github.com/user-attachments/assets/957064b8-a37c-4a76-bd45-82cf1b83786a" />